scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -7,5 +7,5 @@
  * We are allowing a variance of -5 on `MIN_SIZE` and +5 on `MAX_SIZE` to avoid the need for frequent changes, as bundle sizes can fluctuate
  */
 
-export const MIN_SIZE = 673 - 5;
-export const MAX_SIZE = 1213 + 5;
+export const MIN_SIZE = 779 - 5;
+export const MAX_SIZE = 1244 + 5;
```

Comment:
😬

Reply:
chönky

Reply:
I thought it would be smol because I deleted stuff 😿